package.json

```diff
@@ -21,22 +21,23 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@json-schema-tools/meta-schema": "^1.7.0",
-    "@opentelemetry/api": "^1.7.0",
-    "@opentelemetry/exporter-metrics-otlp-proto": "^0.48.0",
-    "@opentelemetry/instrumentation-fastify": "^0.33.0",
-    "@opentelemetry/instrumentation-fetch": "^0.48.0",
-    "@opentelemetry/instrumentation-http": "^0.48.0",
-    "@opentelemetry/instrumentation-pino": "^0.35.0",
-    "@opentelemetry/resources": "^1.21.0",
-    "@opentelemetry/sdk-metrics": "^1.21.0",
-    "@opentelemetry/sdk-node": "^0.48.0",
-    "@opentelemetry/semantic-conventions": "^1.21.0",
+    "@opentelemetry/api": "^1.8.0",
+    "@opentelemetry/exporter-metrics-otlp-proto": "^0.51.0",
+    "@opentelemetry/instrumentation-fastify": "^0.36.0",
+    "@opentelemetry/instrumentation-fetch": "^0.51.0",
+    "@opentelemetry/instrumentation-http": "^0.51.0",
+    "@opentelemetry/instrumentation-pino": "^0.38.0",
+    "@opentelemetry/resources": "^1.24.0",
+    "@opentelemetry/sdk-metrics": "^1.24.0",
+    "@opentelemetry/sdk-node": "^0.51.0",
+    "@opentelemetry/semantic-conventions": "^1.24.0",
     "commander": "^11.0.0",
     "fastify": "^4.23.2",
-    "pino-pretty": "^10.2.3"
+    "pino-pretty": "^10.2.3",
+    "prom-client": "^15.1.2"
   },
   "devDependencies": {
-    "@tsconfig/node18": "^18.2.2",
+    "@tsconfig/node20": "^20.1.4",
```

Comment:
Why did this change? We're still on Node 18 in this project as defined in the .nvmrc file, and the node types below are also at node 18.

I would revert this unless you're intending on upgrading the project to be built on Node 20.